packages/core/src/core/llm-client.ts

```
@@ -76,15 +255,78 @@
+      case "google":
+      case "vertex":
+        response = await generateText({
+          model,
+          prompt,
+          abortSignal: signal,
+          temperature: this.config.temperature,
+          maxTokens: this.config.maxTokens,
+        });
+        break;
+
+      case "mistral":
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)